api/SourceBuffer.json

```diff
@@ -75,7 +75,7 @@
             "version_added": "8"
           },
           "safari_ios": {
-            "version_added": false
+            "version_added": "8"
```

Comment:
I've tested http://software.hixie.ch/utilities/js/live-dom-viewer/saved/8420 on iPhones in BrowserStack to try to confirm when MSE was added, but it's not there even on Safari 13. https://caniuse.com/mediasource suggests that it's only for iPad. If that's correct, we should have notes in the BCD data as well.